message-schemas/generated/jsonschema/ArrowMarker.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
For ease of someone browsing the repo, I would suggest promoting the generated code to the top level as well (i.e. no `generated` folder, just straight `ros1`, `proto`, etc). It's only a few directories.

Reply:
I wanted it to be part of the npm package, and easy to delete & regenerate all at once, so that's why it's inside message-schemas/generated/* currently

Reply:
I still think it would be nice to have this new package move to the top level and replace whatever legacy stuff exists at the top level, and bump the generated stuff up a level so it's easy to find (you can still delete & regenerate it, just need to delete 4 folders instead of 1).

If you really want it in a subfolder, could rename `generated` to `schemas` or something so that it still makes sense for someone browsing the repo.

Reply:
renamed to `schemas`